frontend/web/src/features/guestdashboard/components/GuestFamilyRow.js

```diff
@@ -0,0 +1,191 @@
+              <div style={{ display: "flex", alignItems: "center", gap: 8 }}>
+                <button
+                  type="button"
+                  className="pi-action cancel"
+                  onClick={() => decrease("adults")}
+                  aria-label="Decrease adults"
+                >
+                  
+                </button>
+                <input
+                  className="pi-input"
+                  type="number"
+                  min={0}
+                  max={20}
+                  value={familyCounts.adults}
+                  onChange={(e) =>
+                    setFamilyCounts((prev) => ({
+                      ...prev,
+                      adults: limitBetween(Number(e.target.value) || 0),
+                    }))
+                  }
+                  style={{ width: 90 }}
+                />
+                <button
+                  type="button"
+                  className="pi-action"
+                  onClick={() => increase("adults")}
+                  aria-label="Increase adults"
+                >
+                  <img
+                    src={checkIcon}
+                    alt=""
+                    style={{ visibility: "hidden" }}
+                  />
+                  <span
+                    style={{ position: "absolute", fontSize: 18, lineHeight: 1 }}
+                  >
+                    +
+                  </span>
+                </button>
+              </div>
+            </div>
+
+            {/* Kids */}
+            <div
+              className="booking-details__row"
+              style={{ gridTemplateColumns: "140px auto" }}
+            >
+              <div className="booking-details__label">Kids</div>
+              <div style={{ display: "flex", alignItems: "center", gap: 8 }}>
+                <button
+                  type="button"
+                  className="pi-action cancel"
+                  onClick={() => decrease("kids")}
+                  aria-label="Decrease kids"
+                >
+                  −
+                </button>
+                <input
+                  className="pi-input"
+                  type="number"
+                  min={0}
+                  max={20}
+                  value={familyCounts.kids}
+                  onChange={(e) =>
+                    setFamilyCounts((prev) => ({
+                      ...prev,
+                      kids: limitBetween(Number(e.target.value) || 0),
+                    }))
+                  }
+                  style={{ width: 90 }}
+                />
+                <button
+                  type="button"
+                  className="pi-action"
+                  onClick={() => increase("kids")}
+                  aria-label="Increase kids"
+                >
+                  <img
+                    src={checkIcon}
+                    alt=""
+                    style={{ visibility: "hidden" }}
+                  />
```

Comment:
styling must be in the scss file
```suggestion
                  <img src={checkIcon} alt=""/>
```